js/jquery-scrollspy.js

```diff
@@ -0,0 +1,97 @@
+/*!
+ * jQuery Scrollspy Plugin
+ * Author: @sxalexander
+ * Licensed under the MIT license
+ */
+;(function ( $, window, document, undefined ) {
```

Comment:
Codacy has a fix for the issue: [Unnecessary semicolon.](https://app.codacy.com/gh/myparcelnl/magento/pullRequest?prid=8178215)
```suggestion
(function ( $, window, document, undefined ) {
```